_projects/view/accelerating-your-ultra96/README.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Accelerating Your Ultra96 Developments
+permalink: "/projects/accelerating-your-utra96/"
+description: |-
+    Learn how to use SDSoC to accelerate functions from the processors to the logic in the Ultra96!
+images:
+  - accelerating-your-ultra96.png
+  - mpsoc.png
+categories:
+  - Ultra96
+
+
+---
+# Accelerating Your Ultra96 Developments
+
+
+Learn how to use SDSoC to accelerate functions from the processors to the logic in the Ultra96!
+
+## Project Details
+
+- **Creator:** Adam Taylor
+- **Project Name:** Accelerating Your Ultra96 Developments
+- **Type of Project:** Demonstrations (Projects showcasing individual features of a 96Boards product)
+- **Project Category:** MPSoC, Hardware Acceleration
+- **Board(s) used:** [Ultra96](https://www.96boards.org/product/ultra96/)
```

Comment:
When referencing pages on the same website, please use relative URLs rather than absolute ones. In other words:

```
[Ultra96](/product/ultra96/)
```